git-find-ignored-files.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# git-find-ignored-files.sh
+#
+# Find all files present in the work tree that are ignored by .gitignore
+#
+# Author: Patrick Lühne, https://www.luehne.de/
+#
+
+key="$1"
+
+function print_help
+{
+	echo "usage: git-find-ignored-files [-s | --sort-by-size] [--help]"
```

Comment:
Here is a trick how you could write the help in the script header and print it:
https://github.com/larsxschneider/cached-clone/blob/c2b7ca670da7854e7139e40ce8363cdad3874e8d/git-cclone#L38-L39

Reply:
Thanks for the pointer, I’ll adjust the usage output :+1:.